riva/clients/nmt/riva_nmt_t2t_client.cc

```diff
@@ -270,13 +290,14 @@ main(int argc, char** argv)
           }
       }
     }
+
     auto end = std::chrono::steady_clock::now();
     std::chrono::duration<double> total = end - start;
     LOG(INFO) << FLAGS_model_name << "-" << FLAGS_batch_size << "-" << FLAGS_source_language_code
               << "-" << FLAGS_target_language_code << ",count:" << count
               << ",total time: " << total.count()
               << ",requests/second: " << FLAGS_num_iterations * request_count / total.count()
-              << ",translations/second: " << FLAGS_num_iterations * count / total.count();
+              << ",translations/second: " << total_words/total.count();
```

Comment:
This is not considering number of iterations

Reply:
@ydharavath can you please fix this? basically same number of words are repeated over num_iterations, so need to consider that.